corehq/apps/userreports/static/userreports/js/base.js

```diff
@@ -3,9 +3,9 @@ hqDefine('userreports/js/base', [
     'hqwebapp/js/initial_page_data',
     'reports_core/js/charts',
     'reports_core/js/bootstrap3/maps',
-    'reports/js/bootstrap3/datatables_config',
+    'reports/js/bootstrap5/datatables_config',
```

Comment:
I won't go into detail here since we talked about B5 a bit on another PR, but in short, I'd do any B5 changes separately, for the sake of making both review and QA easier. I'm happy to talk more about the high-level approach to migrating pages to B5 if it's helpful.